contracts/colonyNetwork/IColonyNetwork.sol

```diff
@@ -139,7 +139,18 @@ contract IColonyNetwork is ColonyNetworkDataTypes, IRecovery {
   /// @param _version The version of colony to deploy (pass 0 for the current version)
   /// @param _colonyName The label to register (if null, no label is registered)
   /// @param _orbitdb The path of the orbitDB database associated with the user profile
-  /// @param _useExtensionManager If true, give the ExtensionManager the root role in the colony
+  /// @return colonyAddress Address of the newly created colony
+  function createColony(address _tokenAddress, uint256 _version, string memory _colonyName, string memory _orbitdb)
```

Comment:
If we're cleaning this function up, should we not get rid of `_orbitdb` as well?

Reply:
Is that not being used anymore?

Reply:
It is not. He's only been gone a handful of days, and you've already forgotten Chris' exasperation with Orbit and friends? :stuck_out_tongue:

Reply:
Oh yes, I suppressed that memory.

Reply:
So we still pass it into the ENS reservation, but I'm guessing we can just pass in the empty string?